src/helpers/date.ts

```diff
@@ -0,0 +1,20 @@
+export const getWeekDay = (day: number): string => {
```

Comment:
please use native js methods to get week days

Reply:
ok, changed 
`${new Date(data.current.time).toLocaleDateString('en-US', { weekday: 'long' })}`}`